docker/rbac.providers.azure.Dockerfile

```diff
@@ -12,12 +12,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # -----------------------------------------------------------------------------
-FROM python:3.7
 
-RUN pip install \
-    requests    \
-    rethinkdb   \
-    azure-eventhub
+# -----------------------------------------------------------------------------
+# Begin base docker image config for Hyperledger RBAC Next Directory
+# This should remain the same for all python containers to maximize caching
+# -----------------------------------------------------------------------------
+FROM hyperledger/sawtooth-validator:1.0
```

Comment:
I understand the goal of optimizing caching across containers, but why choose sawtooth-validator as the common base image? Sawtooth validator packages the essentials necessary for consensus (transaction processing, peer communication, etc). How will provider/server modules leverage the consensus boilerplate made available to them?

Reply:
For dev purposes, you pick the image+dockerFile-base that covers the largest number of containers and includes the majority of common prerequisites and go from there. This one covers six containers. Got a better candidate?

Reply:
Why not python:3.X? That should give us the Python bits so we can layer from there.

Reply:
How many containers does that cover? Make a PR and test it.

Reply:
Here is what I am seeing historically.

chatbot: rasa_core
client: node:8
ledger_sync: ubuntu -> sawtooth-validator
rbac_processor: ubuntu -> sawtooth-validator 
rbac.providers.azure: python
rbac.providers.ldap: python -> sawtooth-validator
rbac.server: ubuntu -> sawtooth-validator

It will take some time to test the total build time, but sounds like an interesting experiment. 
Will give it a shot. Also need to confirm no sawtooth deps in server in the process.

Reply:
Created #726